<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SocketTestWindow</class>
 <widget class="QDialog" name="SocketTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sock-test</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>send data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_teSendData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>451</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>recv data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="m_teRecvData">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>451</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>11</y>
     <width>386</width>
     <height>54</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QRadioButton" name="m_rbTcpEnable">
      <property name="text">
       <string>TCP</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>address</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLineEdit" name="m_leAddress">
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QPushButton" name="m_pbListen">
      <property name="text">
       <string>listen(&amp;L)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="m_pbSend">
      <property name="text">
       <string>send(&amp;S)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QRadioButton" name="m_rbUdpEnable">
      <property name="text">
       <string>UDP</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>port:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLineEdit" name="m_lePort">
      <property name="text">
       <string>10240</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="m_pbConnect">
      <property name="text">
       <string>connect(&amp;C)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="4">
     <widget class="QPushButton" name="m_pbDisconnect">
      <property name="text">
       <string>disconnect(&amp;D)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="m_cbHexView">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>250</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>hex display</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
